examples/sql_extended_server_blob_auditing_policies/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "azurerm_resource_group" "test" {
```

Comment:
Please change the alias from "test" to "example" according by hasicorp's comment.